critical-css.js

```diff
@@ -34,6 +34,13 @@ function main(urls) {
 }
 
 function findCriticalCss(cssString, url) {
+  // When a regex is passed from a command line it converts to a string,
+  // so we need to convert it back to a regex
+  const excludeArr = args['exclude'].map((arg) =>
+    arg.startsWith("/") && arg.endsWith("/")
```

Comment:
Would this work if we pass regex flags? ie `/\s/g` or `/\w/gi`

Reply:
Thanks for catching this. I think it will fail to catch if the ending is not `/`. I'll look for a different way to check if it's a regex.

Reply:
Fixed in https://github.com/iFixit/css-gather/pull/9/commits/89f826b2ee03e83ac613e1d0e8d9cd24c9624e47.